src/Credentials/ServiceAccountCredentials.php

```diff
@@ -221,8 +221,8 @@ public function fetchAuthToken(callable $httpHandler = null)
     /**
      * Return the Cache Key for the credentials.
      * For the cache key format is one of the following:
-     * <ClientEmail><Scope[.Sub]>
-     * <ClientEmail><Audience[.Sub]>
+     * ClientEmail.Scope[.Sub]
+     * ClientEmail.Audience[.Sub]
```

Comment:
We should be consistent with our formatting with how we display these cache keys. Here we use `CamelCase`, but everywhere else uses `alligatorCase`. Also, here we have the optional values in `[]`, whereas in the others we don't.

I don't have a strong opinion on which formatting we use (although I like the looks of this one a little better), but we should be consistent.

Reply:
You're right. I changed these formats so many times that I missed some updates, let me fix this.